internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -174,6 +174,7 @@ func resourceKustoCluster() *pluginsdk.Resource {
 						string(kusto.LanguageExtensionNameR),
 					}, false),
 				},
+				DiffSuppressFunc: languageExtensionDiffSuppress,
```

Comment:
we can remove this in favour of making `language_extensions` a Set - this field isn't going to be interpolated, so whilst this is a breaking change, this isn't an issue here.